### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by isapind**
Created a footer component with tests and styling, and have added to App component.

### CODE REVIEW

File: src/components/footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  padding: ${({ theme }) => theme.sizes.spacing.sm};
+
+  & i {
+    font-size: ${({ theme }) => theme.sizes.fonts.xl};
+  }
+
+  & .footer-i-middle {
+    margin-left: ${({ theme }) => theme.sizes.spacing.sm};
+    margin-right: ${({ theme }) => theme.sizes.spacing.sm};
+  }
```
Comment (Si3rr4wow) [inline src/components/footer/Footer.tsx:18]: Just put the same margins on all of them. This was good thinking but you'll regret it it 4 months when product asks us to add a new link to snapchat or something

Comment (isapind) [reply src/components/footer/Footer.tsx]: Thanks Marks, have changed that

**Review by Si3rr4wow — APPROVED** [review]
lgtm! Can you add a ticket to trello tagged with tech debt to incorporate whatever abstraction Khalid comes up with for icons that get lighter on hover please :)

**Comment by isapind** [comment]
Merged + created icon hover ticket on Trello.

**Pull request merged**